### PR TITLE
Add instructions for etcd backup and module deletion

### PR DESCRIPTION
**Pull request by George-Aeillo**
In this PR, I add instructions for etcd backup and module deletion.

### CODE REVIEW

**Review by pfmackin — APPROVED** [review]

File: doc/experimental/phase1/phase1.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Follow these [instructions](https://docs.oracle.com/en/operating-systems/olcne/2
 ocne cluster start --provider none --kubeconfig $KUBECONFIG --auto-start-ui false
 kubectl -n ocne-system rollout status deployment ocne-catalog
 ```
+
+## Perform an ETCD Backup
+Follow these [instructions](../../cluster-management/etcd-backup.md) to backup the ETCD database. 
+These backups should also be done periodically after the migration.
```
Comment (mgianatagh) [inline doc/experimental/phase1/phase1.md:23]: Although this statement is true, I am not sure it is necessary.  There are many things that should be backed up periodically, regardless of a migration.

File: doc/experimental/phase2/phase2.md [hunk 2]
```diff
@@ -77,8 +77,13 @@ After the entire has been upgraded to OCK, then you can safely remove the verraz
...
 ### Delete the Module CRs
-*TDB* Patch the CRs and remove the finalizer
-*TBD* Delete each CR
+There are several CRs of type Module. Once all the nodes have moved to OCK, then they are no longer needed.  
+They can be found by the following kubectl command 
+```text
+kubectl get Module -A
+```
+At this point, the module operator should be removed from the system, so the finalizers have to be deleted. 
+We have instructions for other resources with similar requirements. See https://github.com/oracle-cne/ocne/blob/main/doc/experimental/phase1/oam-remove-objects.md
```
Comment (mgianatagh) [inline doc/experimental/phase2/phase2.md:86]: I think we want to explicitly list the commands to do the patches and remove each CR

File: doc/experimental/phase1/phase1.md (hunk 1 above)
Comment (George-Aeillo) [reply doc/experimental/phase1/phase1.md:23]: Ok!

File: doc/experimental/phase2/phase2.md (hunk 2 above)
Comment (George-Aeillo) [reply doc/experimental/phase2/phase2.md]: Ok!

**Review by mgianatagh — APPROVED** [review]

**Review by pfmackin — APPROVED** [review]

**Pull request merged**